css/index.css

```diff
@@ -70,16 +68,16 @@
      transition: 0.3s ease;
      }
    
-     menu li {
+     .menu li {
      padding: 5px 14px;
      }
      
-     <!--Checkbox-->
-     input [type=checkbox]:checked ~ .menu { 
-     display: inline-block;
+     /*CHECKBOX*/
+     input[type=checkbox] {
+      display:none;
```

Comment:
Is there JS which turns display back on? Maybe a comment to explain the bigger picture.